### ISSUE TITLE
[Bug] Cannot Login

### ISSUE THREAD

**Issue by soulspark666** [toasterofbread/spmp#91]
**Describe the bug**
Cannot log in in the App, tried bot manual and in-app login methog

**Environment**
- OS name and version: Android 13
- SpMp version (include commit hash if you are using a non-release build) : 0.1.3(5)
- Accounts (were you signed in to YouTube, Discord, etc. in the app?): No

**Screenshots**
Manual Login:
![photo1690001277](https://github.com/toasterofbread/spmp/assets/37495396/9e60f5cd-12fa-4a43-8636-74f7031c5f3c)

**Logs**
```
java.lang.NullPointerException
 at com.toasterofbread.spmp.model.YoutubeMusicAuthInfo$Companion$fromHeaders$2.invokeSuspend(Unknown Source:212)
 at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(Unknown Source:8)
 at kotlinx.coroutines.DispatchedTask.run(Unknown Source:100)
 at kotlinx.coroutines.internal.LimitedDispatcher.run(Unknown Source:12)
 at kotlinx.coroutines.scheduling.TaskImpl.run(Unknown Source:2)
 at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(Unknown Source:79)
 Suppressed: coil.network.HttpException: [androidx.compose.ui.platform.MotionDurationScaleImpl@17c2e37, androidx.compose.runtime.BroadcastFrameClock@d7e9fa4, StandaloneCoroutine{Cancelling}@66210d, AndroidUiDispatcher@69c38c2]
```

**Comment by toasterofbread** [comment]
Should be fixed now, please let me know if it works. 

I believe the issue was due to the account being used not being initialised as a YouTube music channel (which is required due to a technical limitation in the app right now), so I was only able to test it properly a few times since I only have so many Google accounts. The account registration dialog I've added will be made optional once I've fixed the limitation I mentioned.

**Comment by soulspark666** [comment]
Hi, sorry for the late reply, I may have missed your response.

SpMp version: 0.1.4(6) [Clean Install]
Login Manually:
![Screenshot_20230910-154104_SpMp](https://github.com/toasterofbread/spmp/assets/37495396/c57d811b-efc1-4779-8185-235bac24cdac)

Login Google sign in:
![Screenshot_20230910-152634_SpMp](https://github.com/toasterofbread/spmp/assets/37495396/2adafec5-e8a9-42aa-b9bc-f738138fc222)
And it crashed.

**Comment by toasterofbread** [comment]
This fix isn't in 0.1.4, you'll have to try a //[recent build](https://github.com/toasterofbread/spmp/actions/workflows/android.yml) or wait for the next release, which I'm planning to publish in a day or two.
